test/src/e2e_vm_tests/test_programs/token_ops_test/Forc.toml

```diff
@@ -0,0 +1,19 @@
+[project]
+author  = "Nick Furfaro"
+license = "MIT"
+name = "token_ops_test"
+entry = "main.sw"
+
+[dependencies]
+std = { git = "http://github.com/FuelLabs/sway-lib-std" }
+core = { git = "http://github.com/FuelLabs/sway-lib-core" }
+test_fuel_coin_abi = { path = "../test_fuel_coin_abi"}
+
+# test_fuel_coin_contract
+[[tx-input]]
+type = "Contract"
+contract-id = "0x16af825dc6a612810b6c80a16f2a7e3248f574f2b0c96babda736fc53d8d8f8c"
+utxo-id = "0xad6aaaa1d6fd78f91693ee2cc124fd43d25bd1c015b88b675ee43d6b5e14058600"
+balance-root = "0xad6aaaa1d6fd78f91693ee2cc124fd43d25bd1c015b88b675ee43d6b5e140586"
+state-root = "0xad6aaaa1d6fd78f91693ee2cc124fd43d25bd1c015b88b675ee43d6b5e140586"
+
```

Comment:
Hmmm. Why does including this not error? It should have been removed in #675. cc @sezna, does including sections that aren't known not result in a error?
```suggestion
```

Reply:
Yeah, just tested and it looks like it ignores unknown stuff. I know rust allows it, too, but provides a warning like "this manifest key doesn't do anything."

Reply:
Filed https://github.com/FuelLabs/sway/issues/694